### DISCUSSION TITLE
Minio bucket counters not updating after reinstallation

### DISCUSSION THREAD

**Discussion by archtechmic** [minio/minio#16675] [Q&A] [answered]
I have a working standalone server deployed on Debian. The Minio storage is mounted with a single disk. There was a need to reinstall Debian. After reinstalling Debian and reinstalling Minio, the service started working. The problem is that the buсkets counters in the web interface and through the mc admin info stopped at numbers before the system was reinstalled and do not change regardless of adding or removing objects to the buсket. Is it possible to solve this problem?
I would appreciate any help

**Comment by klauspost** [comment, marked as answer]
Try upgrading to the latest version. A deadlock in the healing system was fixed there, which can hang the scanner.

**Reply by archtechmic** [reply]
After the update it works. Thank you.
